src/stream_base-inl.h

```diff
@@ -302,9 +308,10 @@ inline void StreamReq::Done(int status, const char* error_str) {
 }
 
 inline void StreamReq::ResetObject(v8::Local<v8::Object> obj) {
-  DCHECK_GT(obj->InternalFieldCount(), StreamReq::kStreamReqField);
+  DCHECK_GT(obj->InternalFieldCount(),
+            StreamReq::kStreamReqField);
```

Comment:
```suggestion
  DCHECK_GT(obj->InternalFieldCount(), StreamReq::kStreamReqField);
```
(no change necessary, right?)